kahuna/public/js/search/query.html

```diff
@@ -75,7 +75,7 @@
             <span class="search__modifier-toggle__text">Sort order</span>
         </button>
 
-        <ul class="search__modifier order__option" ng:class="orderShown ? 'order__option--show' : 'order__option--hide'">
+        <ul ng:hide="searchQuery.filter.orderBy === 'dateAddedToCollection'" class="search__modifier order__option" ng:class="orderShown ? 'order__option--show' : 'order__option--hide'">
```

Comment:
As a rule, shall we use `ng:if` instead of `ng:show`/`ng:hide`? Tends to avoid weird bugs.